A41-xds-rbac.md

```diff
@@ -62,107 +62,134 @@ Supporting the RBAC HTTP filter on server-side leverages [A36: xDS-Enabled
 Servers][A36] and [A39: xDS HTTP Filter Support][A39]. Server-side HTTP filters
 are less common than client-side at this point in time, so this may be the first
 usage of `RouteConfiguration` on server-side and so likely involves adding more
-complete support for A39 by creating+executing filters on server-side and
+complete support for A39 by creating and executing filters on server-side and
 processing `VirtualHost`s and `RoutMatch`es to determine which configuration
 should be provided to each filter. A39 should be consulted for the expected
 behavior.
 
+New validation should occur for `HttpConnectionManager` to allow equating
```

Comment:
Yeah, I was having trouble with that several places. It was really easy to make this stuff too verbose and for it to _still_ not be clear/easy-to-read. I've clarified. I _think_ just mentioning "RBAC" for these first two fields might have been enough, but I went ahead and also qualified the later fields in this paragraph.

I chose to say "RBAC" here instead of using `Principal.direct_remote_ip` because it seemed easier to follow.